presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -532,7 +532,8 @@ public boolean transitionToFailed(Throwable throwable)
         recordDoneStats();
 
         // NOTE: this must be set before triggering the state change, so listeners
-        // can be observe the exception
+        // can be observe the exception.  This is safe because the failure info can
```

Comment:
/s/can be observe the exception.  This / can observe the exception. This

(spacing and be)